src/beignet/_pairwise_displacement.py

```diff
@@ -0,0 +1,29 @@
+from torch import Tensor
+
+
+def pairwise_displacement(Ra: Tensor, Rb: Tensor) -> Tensor:
```

Comment:
Use PyTorch style, e.g., `input` and `other` for binary operators.

Reply:
This is a PyTorch operator so it should be moved to the root module, i.e., `beignet.pairwise_displacement`.